### ISSUE TITLE
Geometry Does Not Exist

### ISSUE THREAD

**Issue by Clenera** [koopjs/koop-sample-app#29]
Got the following error when trying to load the URL in the sample-app:

"Failed to create table Github:benbalter/dc-wifi-social/bars:0 error:error: type \"geometry\" does not exist"

**Comment by ngoldman** [comment]
This may be an issue with the Github provider. https://github.com/koopjs/koop-github

**Comment by jgravois** [comment]
the github provider was behaving correctly when i logged #26 the other day, but i can repro (with koop `2.8.5`).

i'm on a different machine now, but i even tried reverting back to use koop `2.7.2` (which i was pointing at then) and i can repro with that version too.

**Comment by ngoldman** [comment]
If you're using koop-pgcache and postgres, this may be because the PostGIS extension was not successfully added.

@Clenera @jgravois did you remember to create the PostGIS extension for the database you're working with? (instructions here: https://github.com/koopjs/koop-sample-app#instructions, reproduced below).

```
$ createdb koopdev
$ psql koopdev

koopdev=# CREATE EXTENSION postgis;
CREATE EXTENSION
koopdev=# CREATE EXTENSION postgis_topology;
CREATE EXTENSION
koopdev=# CREATE EXTENSION fuzzystrmatch;
CREATE EXTENSION
koopdev=# CREATE EXTENSION postgis_tiger_geocoder;
CREATE EXTENSION
```

`geometry` is (if I recall correctly) a column added by PostGIS.

**Comment by jgravois** [comment]
looking at my database now, it definitely already has the `postgis` extension enabled (not sure what the benefit of the other three are), but since i can no longer repoduce the error, i'm just going to say 'nate was right'

**Comment by ngoldman** [comment]
I've just published alpha versions of koop-gist 2.0.0 and koop-github 2.0.0 that should resolve a fair number of issues. Once they're stable I'll add them to the sample app (which is need of a rewrite as well).

**Comment by ngoldman** [comment]
I've updated the sample app and tested it locally. Everything is working fine as far as I can tell. There have been significant updates to koop-github, koop, and koop-pgcache which should hopefully catch any edge cases.

@Clenera please try cloning the latest version of this repo and ensuring your local PostgreSQL `koopdev` table has the PostGIS extension installed (https://github.com/koopjs/koop-sample-app#instructions).
